.swiftlint.yml

```diff
@@ -0,0 +1,10 @@
+disabled_rules:
+    - identifier_name
+    - type_name
+
+line_length:
```

Comment:
Additionally, I have modified the line_length rule; it is set to 120 by default, but I have increased it to 140